core/src/test/scala/org/apache/spark/util/JsonProtocolSuite.scala

```diff
@@ -1780,7 +1802,19 @@ private[spark] object JsonProtocolSuite extends Assertions {
       |      "Remote Bytes Read" : 0,
       |      "Remote Bytes Read To Disk" : 0,
       |      "Local Bytes Read" : 0,
-      |      "Total Records Read" : 0
+      |      "Total Records Read" : 0,
+      |      "Remote Requests Duration": 0,
+      |      "Push Based": {
```

Comment:
Rename this to "Push Based Shuffle" instead ?
It is within the context of `Shuffle Read Metrics`, but makes it more clear.

Thoughts ?

+CC @otterc, @Ngone51

Reply:
+CC @zhouyejoe

Reply:
Let us do this rename as well. Thoughts ?